sdk/msg/msg.go

```diff
@@ -0,0 +1,53 @@
+package msg
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/stellar/experimental-payment-channels/sdk/state"
+	"github.com/stellar/go/keypair"
+)
+
+type Type int
+
+const (
+	TypeHello           Type = 100
+	TypeOpenRequest     Type = 200
+	TypeOpenResponse    Type = 201
+	TypePaymentRequest  Type = 300
+	TypePaymentResponse Type = 301
+	TypeCloseRequest    Type = 400
+	TypeCloseResponse   Type = 401
+)
+
+type Message struct {
+	Type Type
+
+	Hello *Hello `json:",omitempty"`
```

Comment:
Can we rename "Hello" to something more straightforward? Maybe "Content" would be easier to understand.

Reply:
In protocol that have a handshake or introduction phase, a `Hello` message is a common term to use for the first message in a handshake process where participants introduce themselves. TLS has a ClientHello and ServerHello ([eg](https://datatracker.ietf.org/doc/html/rfc5246#section-7.4.1.2)). Stellar Core has a Hello message ([eg](https://github.com/stellar/stellar-core/blob/ff56e7f207b918aae53710abcc001511ead7ece9/src/xdr/Stellar-overlay.x#L32-L43)). Unlike the other messages there's no request and response for the hello.

Lmk if I've misunderstood your concern.

Reply:
TIL!

This addresses my concern, thanks for explaining.